### PR TITLE
feat: mf-5475 add scroll chain

### PR DESCRIPTION
**Pull request by UncleBill**
https://mask.atlassian.net/browse/MF-5475

### CODE REVIEW

**Comment by guanbinrui** [comment]
Remove the white border part.

image

**Comment by UncleBill** [comment]
> Remove the white border part.
> 
> image

This was intentionally designed. @xiazhiming

**Comment by guanbinrui** [comment]
> > Remove the white border part.
> > image
> 
> This was intentionally designed. @xiazhiming

Not right to me. I see no outline border used in other chain icons. Check here: [packages/web3-shared/evm/src/assets](https://github.com/DimensionDev/Maskbook/tree/develop/packages/web3-shared/evm/src/assets).

**Comment by guanbinrui** [comment]
Please consult @SpaceStation09 to deploy contracts on the scroll chain.

**Review by Jack-Works — DISMISSED** [review]

File: packages/web3-shared/evm/src/constants/chains.json [hunk 1]
```diff
@@ -648,6 +648,32 @@
...
+            "name": "ETH",
+            "symbol": "ETH",
+            "decimals": 18,
+            "logoURL": "https://static.debank.com/image/chain/logo_url/astar/398c7e0014bdada3d818367a7273fabe.png"
+        },
+        "infoURL": "https://blockscout.scroll.io/",
+        "explorers": [
+            {
+                "name": "blockscout",
+                "url": "https://blockscout.scroll.io/",
```
Comment (guanbinrui) [inline packages/web3-shared/evm/src/constants/chains.json]: https://scrollscan.com/

**Review by Jack-Works — DISMISSED** [review]

File: packages/web3-shared/evm/src/constants/chains.json [hunk 2]
```diff
@@ -648,6 +648,32 @@
...
+        "shortName": "Scroll",
+        "rpc": ["https://rpc.scroll.io"],
+        "faucets": [],
+        "features": [],
+        "nativeCurrency": {
+            "chainId": 534352,
+            "name": "ETH",
+            "symbol": "ETH",
+            "decimals": 18,
+            "logoURL": "https://static.debank.com/image/chain/logo_url/astar/398c7e0014bdada3d818367a7273fabe.png"
```
Comment (guanbinrui) [inline packages/web3-shared/evm/src/constants/chains.json]:
```suggestion
            "logoURL": "https://imagedelivery.net/PCnTHRkdRhGodr0AWBAvMA/Assets/blockchains/ethereum/info/logo.png/quality=85"
```

File: packages/web3-shared/evm/src/constants/chains.json [hunk 3]
```diff
@@ -648,6 +648,32 @@
...
+        "name": "Scroll",
+        "network": "mainnet",
+        "fullName": "Scroll",
+        "shortName": "Scroll",
+        "rpc": ["https://rpc.scroll.io"],
+        "faucets": [],
+        "features": [],
+        "nativeCurrency": {
+            "chainId": 534352,
+            "name": "ETH",
```
Comment (guanbinrui) [inline packages/web3-shared/evm/src/constants/chains.json]:
```suggestion
            "name": "Ether",
```

**Review by guanbinrui — DISMISSED** [review]

**Pull request merged**